### PR TITLE
Open Design workshop in Taipei for Open UP

### PR DESCRIPTION
**Pull request by Erioldoesdesign**
Again, I hope this is a good amount of info and also, the right way to do this :D

### CODE REVIEW

File: _posts/2019-30-11-openup-taipei.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: event
+title:  "Open Design @ Open UP global summit: Taipei, Nov 30th - Dec 7th"
+date:   2019-30-11
```
Comment (jancborchardt) [inline _posts/2019-30-11-openup-taipei.md]: Wouldn't it be 2019-11-30? Looks good otherwise. :)

Comment (jancborchardt) [reply _posts/2019-30-11-openup-taipei.md]:
```suggestion
date:   2019-11-30
```

**Review by jancborchardt — APPROVED** [review]
Fixed date format here as well :)

**Comment by jancborchardt** [comment]
Another reviewer maybe @ei8fdb @elioqoshi @bumbleblue? :)

**Review by ei8fdb — APPROVED** [review]
Beautiful! :)

**Review by liveHarshit — APPROVED** [review]

**Pull request merged**